site/docs/v1/tech/core-concepts/applications.adoc

```diff
@@ -8,11 +8,11 @@ description: An overview of FusionAuth Applications
 
 == Overview
 
-A FusionAuth Application is simply something a user can log into. When you use the Login API, you will provide an `applicationId` to indicate what resource you're attempting to obtain authorization.
+A FusionAuth Application is simply something a user can log into. When you use the Login API, you will provide an `applicationId` to indicate what resource you're attempting to obtain authorization. When you use one of the OAuth2 / OpenID Connect authorization grants you will provide a `client_id` in some fashion. This client identifier will be unique to a single FusionAuth Application which allows FusionAuth to verify the User is registered for the requested Application and subsequently return the correct roles.
 
-When you use one of the OAuth2 / OpenID Connect authorization grants you will provide a `client_id` in some fashion. This client identifier will be unique to a single FusionAuth application which allows FusionAuth to verify the User is registered for the requested Application and subsequently return the correct roles.
+A FusionAuth Application holds configuration for how your applications interact with FusionAuth. A one to one mapping between an external web, mobile or desktop application and a FusionAuth Application is not required. You could have multiple web and mobile applications configured to use one FusionAuth Application. You could have one web application authenticate against multiple FusionAuth Applications, if different authentication scenarios were wrapped up in one application (this isn't very common, though).
```

Comment:
I had to read this twice before I realized you were describing two different scenarios.  If you wanted you could say in the last sentence "In another scenario..." or "You could also..." or something like that.